products/rocket-chat.md

```diff
@@ -21,9 +21,16 @@ identifiers:
 -   repology: rocketchat-server
 
 # https://docs.rocket.chat/docs/rocketchat-release-notes
-# eoas(x) = releaseDate(x)
+# eoas(x) = releaseDate(x) (there is no more an active support phase, kept for historical purpose)
 # eol(x) can be found on https://docs.rocket.chat/docs/rocketchat-release-notes
 releases:
+-   releaseCycle: "7.6"
+    releaseDate: 2025-05-13
+    eoas: 2025-05-13
```

Comment:
```suggestion
    eoas: 2025-11-30 # lets use eol date for its eoas instead of its releasedate
```

Reply:
The doc says:

> Each release of Rocket.Chat is supported for six months after its release date (...). During the support window, Rocket.Chat focuses on delivering security patches and responding to support queries concerning configuration and usage for the versions specified. Any bugs that are not security-related will be addressed solely in future releases.

So the support is focused on security support only. This is not what can be called "active support".